.github/ISSUE_TEMPLATE/kubernetes_bump.md

```diff
@@ -21,20 +22,12 @@ This section contains tasks to update our book, e2e testing and CI to use and te
 as well as changes to Cluster API that we might have to make to support the new Kubernetes version. All of these
 changes should be cherry-picked to all release series that will support the new Kubernetes version.
 
-* [ ] Modify quickstart and CAPD to use the new Kubernetes release:
-  * Bump the Kubernetes version in:
-    * `test/*`: search for occurrences of the previous Kubernetes version
-    * `Tiltfile`
-  * Ensure the latest available kind version is used (including the latest images for this kind release)
-    * Add new images in the [kind mapper.go](https://github.com/kubernetes-sigs/cluster-api/blob/0f47a19e038ee6b0d3b1e7675a62cdaf84face8c/test/infrastructure/kind/mapper.go#L79).
-      * See the [kind releases page](https://github.com/kubernetes-sigs/kind/releases) for the list of released images.
-    * Set new default image for the [test framework](https://github.com/kubernetes-sigs/cluster-api/blob/0f47a19e038ee6b0d3b1e7675a62cdaf84face8c/test/framework/bootstrap/kind_provider.go#L40)
-    * If code changes are required for CAPD to incorporate the new Kind version, update [kind latestMode](https://github.com/kubernetes-sigs/cluster-api/blob/0f47a19e038ee6b0d3b1e7675a62cdaf84face8c/test/infrastructure/kind/mapper.go#L66)
-    * Prior art: #10094
-  * Verify the quickstart manually
-  * Bump `InitWithKubernetesVersion` and `WorkloadKubernetesVersion` in `clusterctl_upgrade_test.go`
-    * Note: Only bump for Cluster API versions that will support the new Kubernetes release.
-  * Prior art: #9160
+* [ ] Modify CAPD to use a release candidate of the upcoming Kubernetes release:
+  * Bump the Kubernetes version in `test/*`.
```

Comment:
Hm maybe we should be more precise (because we shouldn't bump the version in kind_mapper / kind_provider.go.

Should we add something like (except ....)